addons/point_of_sale/models/pos_config.py

```diff
@@ -39,6 +39,16 @@ class PosConfig(models.Model):
     _name = 'pos.config'
     _description = 'Point of Sale Configuration'
 
+    def _get_fields_to_check(self):
+        return [
+            {'field': 'active',                 'name': _('archive')},
```

Comment:
this can probably be simplified. No need for a list of dict. Either use a dict with the name of the field as the key or even a list is probably enough (and use `self._fields[key].get_description(self.env)['string']` to get the label).